emails/tests/fixtures/dmarc_failed_email_sns_body.json

```diff
@@ -3,7 +3,7 @@
p 2021 13:16:39 -0700 (PDT)\"},{\"name\":\"DKIM-Signature\",\"value\":\"v=1; a=rsa-sha256; c=relaxed/relaxed; d=mozilla.com; s=google; h=mime-version:references:in-reply-to:from:date:message-id:subject:to; bh=QVQjY8EQBY32Lz1hQpUNIWkRzG2V90LV6NjJIFqNgeI=; b=Vr+QfF7irjzafmpzKSx5DqnQg+HPcyN7qYf5mLWFENnj879ounkR7S2rvPrOZbXcESFK9lbNRwgRU4ZbsvF3UW+GAvpTD+sx3+O+6VDc9JD96t9N5aNsNrhbr9xuwh6VfPAp6mD4if+5U+DrSS+ap1Q/j0KiqqpB92buQPaFkBg=\"},{\"name\":\"X-Google-DKIM-Signature\",\"value\":\"v=1; a=rsa-sha256; c=relaxed/relaxed; d=1e100.net; s=20210112; h=x-gm-message-state:mime-version:references:in-reply-to:from:date :message-id:subject:to; bh=QVQjY8EQBY32Lz1hQpUNIWkRzG2V90LV6NjJIFqNgeI=; b=rVU4pvR/qkIhB3wGYFquOe6ci5ukQygQ59GD3NBdS68aIViM+PHfaab/5D0to1rm5/ EW83dF8mesxJjPZHO8FAhvnReHy2+EpkCb5LHhiZxN13/DG5ZMdoFt5x8Utqxmn8p1hv MqhTybZSwx0+k5jv/LhCaM5HioWNF3dB0igqUzkRKnj2anWOk6YaahLY0fvTxi63+zC9 NLxnsDTJo9PKktnJYpjfCcNJeLg1JoHxJ7+0ZW5nAasx/GqjkKLtFol/DFxtxWgsd8MW 57cXAr45fp/AGBcz5Syszmukf8tcV69jOCjoGCKzNZjAIeSdCpbfJEWta9OdsAEzCAQB sQpA==\"},{\"name\":\"X-Gm-Message-State\",\"value\":\"AOAM530JbDyes6rj2oaKIiqplpvmPGUt287IUVP8fBqUSOZeh7zx4dVx IANrBrVc5Je1vtoDbEiutEXEFNLkqM/UlbBvnzE+TSrvilI=\"},{\"name\":\"X-Google-Smtp-Source\",\"value\":\"ABdhPJyQyz1BnryA8UWbfuDa0hdY7nfi1hSLkgvxPW5+DsgPpnAJ6wcyaPkpw7w3ld7DPbMiOWFM454pNzd9BFGxuU8=\"},{\"name\":\"X-Received\",\"value\":\"by 2002:a1c:417:: with SMTP id 23mr949637wme.161.1632773798173; Mon, 27 Sep 2021 13:16:38 -0700 (PDT)\"},{\"name\":\"MIME-Version\",\"value\":\"1.0\"},{\"name\":\"In-Reply-To\",\"value\":\"<CA+J4FJFw0TXCr63y9dGcauvCGaZ7pXxspzOjEDhRpg5Zh4ziWg@mail.gmail.com>\"},{\"name\":\"From\",\"value\":\"Sender Name <source@sender.com>\"},{\"name\":\"Date\",\"value\":\"Mon, 27 Sep 2021 15:16:26 -0500\"},{\"name\":\"Message-ID\",\"value\":\"<CA+J4FJFjMP7djF0ph8f_pPQg6FvCu_vWuUbG2QxtjPtikvYWXw@mail.gmail.com>\"},{\"name\":\"Subject\",\"value\":\"Re: Test Mozilla User New Domain Address\"},{\"name\":\"To\",\"value\":\"sender@test.com\"},{\"name\":\"Content-Type\",\"value\":\"multipart/alternative; boundary=\\\"0000000000003d386c05ccffc614\\\"\"}],\"commonHeaders\":{\"returnPath\":\"source@sender.com\",\"from\":[\"Sender Name <source@sender.com>\"],\"date\":\"Mon, 27 Sep 2021 15:16:26 -0500\",\"to\":[\"sender@test.com\"],\"messageId\":\"<CA+J4FJFjMP7djF0ph8f_pPQg6FvCu_vWuUbG2QxtjPtikvYWXw@mail.gmail.com>\",\"subject\":\"Re: Test Mozilla User New Domain Address\"}},\"receipt\":{\"timestamp\":\"2021-09-27T20:16:39.558Z\",\"processingTimeMillis\":1086,\"recipients\":[\"sender@test.com\"],\"spamVerdict\":{\"status\":\"PASS\"},\"virusVerdict\":{\"status\":\"PASS\"},\"spfVerdict\":{\"status\":\"FAIL\"},\"dkimVerdict\":{\"status\":\"PASS\"},\"dmarcVerdict\":{\"status\":\"FAIL\"},\"dmarcPolicy\":\"reject\",\"action\":{\"type\":\"S3\",\"topicArn\":\"arn:aws:sns:us-east-1:927034868273:fxprivaterelay-SES-processor-topic\",\"bucketName\":\"test-bucket\",\"objectKeyPrefix\":\"/emails\",\"objectKey\":\"/emails/objectkey123\"}}}",
```

Comment:
How did you make the changes to these fixtures? Is it from a live email or manual guesses?

Reply:
I made manual changes.